package.json

```diff
@@ -141,5 +141,8 @@
       "cliPath": "node_modules/commitizen",
       "path": "./scripts/cz-adapter"
     }
+  },
+  "dependencies": {
+    "node-sass": "^4.13.1"
```

Comment:
Could you please remove this dependency? I know it's the one that was causing you problems at the beginning but I think it's not necessary here.

Reply:
will do 😄